tests/test_packages/test_skills/test_carpark.py

```diff
@@ -182,52 +80,23 @@ def test_carpark(self, pytestconfig):
 
         with open("aea-config.yaml", "w") as f:
             f.write(whole_file)
-
-        os.chdir(self.t)
+        # agent config update block end
 
         # Setup Agent two
-        agent_two_dir_path = os.path.join(self.t, self.agent_name_two)
+        agent_two_dir_path = os.path.join(self.t, agent_name_two)
         os.chdir(agent_two_dir_path)
 
-        result = self.runner.invoke(
-            cli,
-            [*CLI_LOG_OPTION, "add", "--local", "connection", "fetchai/oef:0.2.0"],
-            standalone_mode=False,
-        )
-        assert result.exit_code == 0
-
-        result = self.runner.invoke(
-            cli,
-            [
-                *CLI_LOG_OPTION,
-                "config",
-                "set",
-                "agent.default_connection",
-                "fetchai/oef:0.2.0",
-            ],
-            standalone_mode=False,
-        )
-        assert result.exit_code == 0
-
-        result = self.runner.invoke(
-            cli,
-            [
-                *CLI_LOG_OPTION,
-                "add",
-                "--local",
-                "skill",
-                "fetchai/carpark_client:0.1.0",
-            ],
-            standalone_mode=False,
-        )
-        assert result.exit_code == 0
-
-        result = self.runner.invoke(
-            cli, [*CLI_LOG_OPTION, "install"], standalone_mode=False
-        )
-        assert result.exit_code == 0
-
-        # Load the agent yaml file and manually insert the things we need
+        self.add_item("connection", "fetchai/oef:0.1.0")
+        self.add_item("skill", "fetchai/carpark_client:0.1.0")
+        self.run_install()
+
+        # TODO: same as above. Remove this block and replace with next two commented lines
+        # when "aea config set" will be able to handle dictionaties and non-existing keys
+
+        # setting_path = "agent.ledger_apis.fetchai.network"
+        # self.set_config(setting_path, "testnet")
+
+        # agent config update block start
         file = open("aea-config.yaml", mode="r")
```

Comment:
This logic from lines 100-118 seems suitable for extraction to the parent class

Reply:
with find_text, replace_text as the kwargs, so they stay here and are fed to the replace method